sbncode/DetSim/FilterSimEnergyDeposits_module.cc

```diff
@@ -50,55 +41,38 @@ class FilterSimEnergyDeposits : public art::EDProducer {
   // Declare member data here.
 
   art::InputTag fInitSimEnergyDepositLabel;
-  std::string fOutSimEnergyDepositLabel;
-  double fP1_x;
-  double fP1_y;
-  double fP1_z;
-  double fP2_x;
-  double fP2_y;
-  double fP2_z;
-  static constexpr auto& kModuleName = "FilterSimEnergyDeposit";
-
-  bool isWithinVolume(TVector3 p1, TVector3 p2, TVector3 sedep);
-  double ShiftX(double z);
+  geo::BoxBoundedGeo fBox;
+  static constexpr auto kModuleName = "FilterSimEnergyDeposit";
+  const double ShiftX(double z);
```

Comment:
This should be:
```suggestion
  double ShiftX(double z) const;
```
The former promises to return a "constant" value, while the latter promises that it's not going to change anything in the whole object (that is, treat it like `const FilterSimEnergyDeposits`).